src/cli/services/__tests__/linter.test.ts

```diff
@@ -669,23 +653,7 @@ describe('Linter service', () => {
         }),
         expect.objectContaining({
           code: 'oas2-schema',
-          message: '`description` property type should be string.', // this is covered by 'info-description' as well
-          path: ['description'],
-          range: {
-            end: {
-              character: 18,
-              line: 2,
-            },
-            start: {
-              character: 17,
-              line: 2,
-            },
-          },
-          source: expect.stringContaining('__tests__/__fixtures__/refs/contact.json'),
-        }),
```

Comment:
This is new since I rebased on the latest develop and I can't yet explain why this message is filtered out because of my change.